src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/OffenderNonAssociationRepository.kt

```diff
@@ -20,7 +20,7 @@ interface OffenderNonAssociationRepository :
          from OffenderNonAssociation na, Offender o1, Offender o2 
```

Comment:
Comment refers to nomsId but we now use offenderId

Reply:
whoops yes, will fix next time